implementations/rust/ockam/ockam_command/src/project/util.rs

```diff
@@ -299,6 +268,58 @@ pub async fn check_project_readiness(
         spinner.finish_and_clear();
     }
 
+    Ok(project)
+}
+
+pub async fn check_project_readiness(
+    ctx: &ockam::Context,
+    opts: &CommandGlobalOpts,
+    cloud_opts: &CloudOpts,
+    api_node: &str,
+    tcp: Option<&TcpTransport>,
+    mut project: Project,
+) -> Result<Project> {
+    // Total of 10 Mins sleep strategy with 5 second intervals between each retry
+    let total_sleep_time_ms = 10 * 60 * 1000;
+    let retry_strategy = FixedInterval::from_millis(5000).take(total_sleep_time_ms / 5000);
```

Comment:
If you do take the time to make the previous sleep time a const; that can be done here too

Reply:
Done in https://github.com/build-trust/ockam/pull/4781/commits/4f5c913a0fd556a2614eb0989febe0891a8e301e